config.lua

```diff
@@ -164,8 +164,13 @@ function BerserkAddon:DetectDeadPlayer()
 end
 
 function BerserkAddon:ChatCommand(input)
+
     if not input or input:trim() == "" then
         InterfaceOptionsFrame_OpenToCategory(self.optionsFrame)
+    elseif input == "help" or input == "?" then
+        BerserkAddon:printHelp();
+    elseif input == "mr" or input == "mage rotation" then
+        BerserkAddon:prettyMageRotation();
```

Comment:
We are supposed to handle slash commands using the `HandleCommand` method.
if you take a look at the `else` statement it is calling the `BerserkAddon` with the argument `input`.

I tried a few variations of this code trying to make this work using this library like this example https://www.wowace.com/projects/ace3/pages/api/ace-config-cmd-3-0 but it didnt work for me

I am not sure how to handle slash commands with this library. Your approach is actually really good ty

Reply:
ya the "input" variable is just all the junk after your slash command so I just applied a basic approach here to parse those args. There is probably a nicer way to do it, and as we add more lua lacks a real ```switch() {...}``` so this if else if becomes huge